rel/overlay/etc/default.ini

```diff
@@ -931,6 +934,11 @@ url = {{nouveau_url}}
 ;max_idle = 600000
 ;enable = true
 
+[quickjs]
+; Memory limit in bytes. Default is undefined and so the built-in C default
+; of 64MB is used
+;memory_limit_bytes = 67108864
```

Comment:
If the default goes after the equal sign, shouldn't that also be 64MB or `64000000`?

Reply:
It's power of two MB version (I never remember if it's MiB or MB). But in this case it just matches the  `DEFAULT_STACK_SIZE (64L * 1024L * 1024L)` define from [couchjs.c](https://github.com/apache/couchdb/blob/qjs/src/couch_quickjs/c_src/couchjs.c#L20).